ddtrace/tracer/traceparent.go

```diff
@@ -0,0 +1,15 @@
+package tracer
+
+import (
+	"fmt"
+	"github.com/signalfx/signalfx-go-tracing/ddtrace"
+)
+
+func FormatAsTraceParent(context ddtrace.SpanContext) (string,bool) {
```

Comment:
Do we expect to return invalid traceparent strings from this func? If not, may be get rid of the bool return val and just return a single string? Caller can then check for `traceParent != ""` instead of `ok`.

Reply:
The only case it would be "invalid" would be if the spanid/traceid are 0.  I thought about using "" as the "not a real traceparent" but thought it wasn't very idiomatic go.  I like your cleanup of the caller.

Reply:
Hmm. I think it actually _might_ be idiomatic. I don't know. Generally, a second boolean return value is returned by operations that can't distinguish between a "not found" vs "found but was empty string" situations like a map lookup. Other cases generally tend to use a single return value.